spec/Task/AtoumSpec.php

```diff
@@ -88,8 +89,14 @@ function it_runs_the_suite(ProcessBuilder $processBuilder, Process $process, Con
         $result->isPassed()->shouldBe(true);
     }
 
-    function it_throws_exception_if_the_process_fails(ProcessBuilder $processBuilder, Process $process, ContextInterface $context)
-    {
+    function it_throws_exception_if_the_process_fails(
+        ProcessBuilder $processBuilder,
+        Process $process,
+        ContextInterface $context,
+        ProcessFormatterInterface $formatter
+    ) {
+        $formatter->format($process)->willReturn(Argument::type('string'));
```

Comment:
Lets avoid using the `Argument::type(...)` in a will* statement.
This only works because there is a `\Prophecy\Argument\Token\TypeToken::__toString()` method.
If you want to use a diferent type like integer, you'll get:

```
Argument::type('integer')
 notice: Object of class Prophecy\Argument\Token\TypeToken could not be converted to int
```

I prefer to just return a string instead. (This one is being used all over the newly introduces spec changes)